packages/rpc-graphql/src/context.ts

```diff
@@ -0,0 +1,10 @@
+import { SolanaRpcMethods } from '@solana/rpc-core';
+import { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+
+export interface RpcGraphQLContext {
+    rpc: Rpc<SolanaRpcMethods>;
+}
+
+export function createSolanaGraphQLContext(rpc: Rpc<SolanaRpcMethods>): RpcGraphQLContext {
+    return { rpc };
+}
```

Comment:
What's the purpose of this indirection? Eg why don't we just have `rpc` instead of `context: { rpc }` in `RpcGraphQL`? Do you add more to `RpcGraphQLContext` in future PRs?